src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -57,13 +62,15 @@ public class TwilioRestClient : ITwilioRestClient
         /// <param name="region">region to make requests for</param>
         /// <param name="httpClient">http client used to make the requests</param>
         /// <param name="edge">edge to make requests for</param>
+        /// <param name="userAgentExtensions">Additions to the user agent string</param>
         public TwilioRestClient(
             string username,
             string password,
             string accountSid = null,
             string region = null,
             HttpClient httpClient = null,
-            string edge = null
+            string edge = null,
+            string[] userAgentExtensions = null
```

Comment:
I don't think this should be added to the constructor (I know, we've done it before) as it technically breaks the interface of a dynamically linked library: https://stackoverflow.com/questions/30317625/does-adding-optional-parameters-change-method-signatures-and-would-it-trigger-me

Reply:
So will the user agent extensions be set through the request object? 
E.g. ` testRequest.UserAgentExtensions = userAgentExtensions;`